lib/command/Command.js

```diff
@@ -132,6 +137,17 @@ class Command {
         return !req;
     }
 
+    cooldownCheck(userID) {
+        if(this.usersOnCooldown.has(userID)) {
+            return false;
+        }
+        this.usersOnCooldown.add(userID);
+        setTimeout(() => {
+            this.usersOnCooldown.delete(userID);
+        }, this.cooldown);
```

Comment:
Shouldn't the three lines above only be set on a successful command? (promise after execute?)
It doesn't make sense to set them here during the check stage since the command could fail and the user would still be on cooldown.